src/includes/getting-started-config/node.mdx

```diff
@@ -1,9 +1,29 @@
-You need to inform the Sentry Node SDK about your DSN:
+Once this is done, Sentry's Node SDK captures all unhandled exceptions and transactions.
 
-```javascript
-const Sentry = require("@sentry/node");
+```javascript {tabTitle:ESM}
+import * as Sentry from "@sentry/node";
+import * as Tracing from "@sentry/tracing";
 // or use es6 import statements
 // import * as Sentry from '@sentry/node';
```

Comment:
These 2 commented lines can be removed, as we have separate `esm/commonjs` tabs now.

Reply:
👍